### PR TITLE
Convert search to command palette

### PR DESCRIPTION
**Pull request by geowrgetudor**
Hi Dries. This resolves [#834](https://github.com/laravelio/laravel.io/issues/834)

Fyi, search on mobile is broken on the live version.
I'd ditch the articles/threads snippets and I'd leave only the titles. What do you think about it?

image

image

image

image

### CODE REVIEW

**Comment by driesvints** [comment]
Amazing! that's incredible work @geowrgetudor 👏

I'll try to review this as soon as I'm able.

File: resources/views/_partials/_search.blade.php [hunk 1]
```diff
@@ -65,6 +142,4 @@ class="border-0 border-t border-b block pl-10 border-gray-300 md:border md:round
             <a href="https://algolia.com" class="flex justify-end px-4 py-2">
                 <img src="{{ asset('images/algolia.svg') }}" class="h-4 mx-4 my-2" />
             </a>
-        </div>
-    </div>
-</div>
+        </div> --}}
```
Comment (driesvints) [inline resources/views/_partials/_search.blade.php:145]: Why did you comment this out?

**Comment by driesvints** [comment]
This lgtm. Just one question about commented out code.

File: resources/views/_partials/_search.blade.php (hunk 1 above)
Comment (geowrgetudor) [reply resources/views/_partials/_search.blade.php:145]: That's a part from the old code. I forgot to delete it. I'll fix it tomorrow. :)

**Comment by driesvints** [comment]
@geowrgetudor thank you so much for this one! :)

**Comment by geowrgetudor** [comment]
We should exclude "/" trigger while the editor is focused. Didn't cross my mind when I was implementing it, but I just stumbled upon this while typing an URL path into a reply form. I'll look into it.

**Pull request merged**